Automatically create command when plugin is loaded

avionix-g opened the pull request:
This addition obviates the need to run `command! -range=% -nargs=1 Align lua require'align'.align(<f-args>)` separately. Instead, it runs automatically when the plugin is loaded.

avionix-g committed:
Create plugin/align.vim

RRethy reviewed (changes requested):
Docs have not been updated and there should be a command guard that wraps the command definition to avoid overwriting the user's config.

avionix-g committed:
Add command for align_lines; change line arrangement

Adjust arg count for AlignLines command definition

Set -nargs to + for AlignLines

Convert strings (passed from Vim command) to int

avionix-g committed:
Update docs from align.align to Align and AlignLines

Update docs with example

Added more examples to docs

Update README

avionix-g committed:
Convert to start-inclusive range

avionix-g commented:
I've made a couple changes:
- Documentation has been updated and examples provided
- Added a command for align_lines()
- Changed align_lines() to work with a start-inclusive range. That way the arguments correspond directly to line numbers. If this change is out of the scope of this PR/not desired, let me know.

I'm not sure I understand what you mean by a command guard wrapping the  command definition. What would that look like?

RRethy commented:
No idea why some of those changes are added, not at all related to the description of the PR.
